app/scripts/components/datasets/s-usage/index.js

```diff
@@ -45,11 +45,18 @@ function DatasetsUsage() {
       <PageMainContent>
         <PageHero title={`${dataset.data.name} Usage`} />
         <FoldProse>
-          {datasetUsage?.url && datasetUsage?.title ? (
+          {datasetUsages?.length ? (
             <>
               <p>
-                Check out how to use this dataset in this example notebook:{' '}
-                <a href={datasetUsage.url}>{datasetUsage.title}</a>.
+                Check out how to use this dataset:{' '}
+                <ul>
+                  {datasetUsages.map((datasetUsage) => (
+                    <li key={datasetUsage.url}>
+                      {datasetUsage.description}:{' '}
+                      <a href={datasetUsage.url}>{datasetUsage.title}</a>
+                    </li>
+                  ))}
+                </ul>
               </p>
```

Comment:
A list can't be a children of `p`. Move it after the closing tag

Reply:
Done in https://github.com/NASA-IMPACT/delta-ui/pull/311/commits/3db9861271a2b0eac3a1a0db1acf827916d6c8d3